### DISCUSSION TITLE
Best Practice for Voice Conversion with FreeVC

### DISCUSSION THREAD

**Discussion by etlweather** [idiap/coqui-ai-TTS#76] [General]
What are the best practices to get the best voice conversion (converting an existing audio file to sound like a given target audio file)?

1. Is it better to give longer target audio (`--target_wav`), or shorter? 
2. Is there a preferred input audio format? WAV 16bit, 24bit? 48kHz, 44kHz?
4. Should I fine-tune `freevc24`? How much data and how to format the data for best results?
5. Or should I train from scratch?
4. What is the memory requirements? The `freevc24` model isn't big but somehow, this task takes more than 24GB of VRAM on GPU and on CPU it seems to take quite a bit too, like 30GB. Does it need more memory if I have longer audio input?

**Comment by eginhard** [comment]
1. Try it out
2. FreeVC uses 16 kHz input, anything else will be resampled automatically
3. and 4. The FreeVC training code was never released
